benches/tests.rs

```diff
@@ -3,59 +3,60 @@ extern crate bencher;
 
 use bencher::Bencher;
 use onecode::ser as one_code_ser;
+use parity_scale_codec::Encode;
 
-use std::collections::HashMap;
-#[derive(serde::Serialize)]
-struct StructToSerialize {
-    boolean: bool,
-    positive_integer: u8,
-    negative_integer: i16,
-    positive_float: f32,
-    negative_float: f32,
-    negative_float_comma: f32,
-    empty_string: String,
-    number_string: String,
-    latin_string: String,
-    cyrillic_string: String,
-    japan_string: String,
-    number_list: Vec<u16>,
-    string_list: Vec<String>,
-    number_dictionary: HashMap<String, u16>,
-    string_dictionary: HashMap<String, String>,
-    list_dictionary: HashMap<String, Vec<u16>>,
-    null: Option<()>,
-}
-
-fn serialize(bench: &mut Bencher) {
-    let mut number_dictionary = HashMap::new();
-    number_dictionary.insert("1".to_string(), 1);
-    let mut string_dictionary = HashMap::new();
-    string_dictionary.insert("1".to_string(), "1".to_string());
-    let mut list_dictionary = HashMap::new();
-    list_dictionary.insert("1".to_string(), vec![1, 2, 3]);
+pub fn get_test_struct() -> StructToSerialize{
```

Comment:
Let's replace it with `StructToSerialize::new()` function in `impl StructToSerialize`